app/Libraries/UKCP.php

```diff
@@ -3,31 +3,44 @@
 namespace App\Libraries;
 
 use App\Models\Mship\Account;
+use Bugsnag\BugsnagLaravel\Facades\Bugsnag;
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\ClientException;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Psr\Http\Message\ResponseInterface;
 
 class UKCP
```

Comment:
Might be worth creating a singleton in the AppServiceProvider to auto-resolve the dependency injection

Reply:
Did this originally, it doesn't make a difference. The dependency is automatically resolved anyway

Reply:
that would be the major motivation behind it